src/coreclr/System.Private.CoreLib/src/System/StubHelpers.cs

```diff
@@ -508,6 +508,17 @@ internal static unsafe void ConvertToNative(string? strManaged, IntPtr nativeHom
             managed.Slice(0, numChars).CopyTo(native);
             native[numChars] = '\0';
         }
+
+        internal static unsafe string ConvertToManaged(IntPtr nativeHome, int length)
```

Comment:
This looks slightly different than 

https://github.com/dotnet/runtime/blob/1c81556da6058abe2fe2d9c9e96a58933af980fe/src/coreclr/System.Private.CoreLib/src/System/StubHelpers.cs#L184-L193

Should be consolidate on the pattern?

Reply:
I can switch this method to that pattern. I can't move that one to this pattern as cleanly since there's no constructor for `string` that takes a `ReadOnlySpan<sbyte>`.